src/ast_utils.h

```diff
@@ -18,6 +18,7 @@
 #define wasm_ast_utils_h
 
 #include "wasm.h"
+#include "wasm-traversal.h"
```

Comment:
It has not only the visitor, which visits one node, but the walker, which walks a whole AST. And we should add more walkers (pre-order, in order, etc.). "Traversal" seems to cover both visiting and walking?